cpp/src/twisterx/python/net/comm/Callback.cpp

```diff
@@ -0,0 +1,20 @@
+//
+// Created by vibhatha on 4/27/20.
+//
```

Comment:
cpp file names should start with simple letter, please remove Created by vibhatha header generated by clion

Reply:
modified